tests/manual/testpage.html

```diff
@@ -110,6 +110,9 @@
     <div>
       <a href="testwindowopen.html">window open test page</a>
     </div>
+    <div>
+      <a href="popup-test-html">popup test page</a>
```

Comment:
```suggestion
      <a href="popup-test.html">popup test page</a>
```
Other test pages are starting with "test" => worth to align this to that naming testpopup.html. There are some subtests that are not suffixed with test.